OpenRA.Game/Player.cs

```diff
@@ -161,16 +163,24 @@ public Player(World world, Session.Client client, PlayerReference pr)
 				DisplayFaction = ChooseDisplayFaction(world, pr.Faction);
 			}
 
+			IsBot = BotType != null;
```

Comment:
I assume this is needed by something called via `INotifyCreated` - can we please have a comment explaining this?

Reply:
Otherwise, moving this between the big comment below and the `var playerActorType` might make it clearer by context.